tests/e2e/e2e.json

```diff
@@ -1 +1 @@
oad"},{"path":"01-beacon-type","file":"00-resourcetiming-disabled"},{"path":"01-beacon-type","file":"01-resourcetiming-enabled-browser-supports"},{"path":"01-beacon-type","file":"02-resourcetiming-enabled-browser-unsupported"},{"path":"02-snippet","file":"00-snippet"},{"path":"03-load-order","file":"00-before-page-load"},{"path":"03-load-order","file":"01-after-page-load"},{"path":"03-load-order","file":"02-after-page-load-tag-manager"},{"path":"05-angular","file":"00-simple"},{"path":"05-angular","file":"03-ng-app"},{"path":"05-angular","file":"04-route-change"},{"path":"05-angular","file":"05-route-change-hashtags"},{"path":"05-angular","file":"06-hard-nav-resources"},{"path":"05-angular","file":"07-soft-nav-resources"},{"path":"05-angular","file":"08-no-images"},{"path":"05-angular","file":"09-autoxhr-after-load"},{"path":"05-angular","file":"10-autoxhr-overlapping"},{"path":"05-angular","file":"11-autoxhr-trigger-additional"},{"path":"05-angular","file":"12-autoxhr-trigger-additional-after-delay"},{"path":"05-angular","file":"13-autoxhr-disabled"},{"path":"05-angular","file":"14-autoxhr-before-page-load"},{"path":"05-angular","file":"15-delayed-boomerang"},{"path":"05-angular","file":"16-autoxhr-after-load-with-mutation"},{"path":"06-bugs","file":"92560"},{"path":"06-bugs","file":"95726"},{"path":"07-autoxhr","file":"00-xhrs"},{"path":"07-autoxhr","file":"01-img-src-change"},{"path":"07-autoxhr","file":"02-onclick"},{"path":"08-ember","file":"00-simple"},{"path":"08-ember","file":"04-route-change"},{"path":"08-ember","file":"05-route-change-hashtags"},{"path":"08-ember","file":"06-hard-nav-resources"},{"path":"08-ember","file":"07-soft-nav-resources"},{"path":"08-ember","file":"08-no-images"},{"path":"08-ember","file":"09-autoxhr-after-load"},{"path":"08-ember","file":"10-autoxhr-overlapping"},{"path":"08-ember","file":"11-autoxhr-trigger-additional"},{"path":"08-ember","file":"12-autoxhr-trigger-additional-after-delay"},{"path":"08-ember","file":"13-autoxhr-disabled"},{"path":"08-ember","file":"14-autoxhr-before-page-load"},{"path":"08-ember","file":"15-delayed-boomerang"},{"path":"09-backbone","file":"00-simple"},{"path":"09-backbone","file":"04-route-change"},{"path":"09-backbone","file":"05-route-change-hashtags"},{"path":"09-backbone","file":"06-hard-nav-resources"},{"path":"09-backbone","file":"07-soft-nav-resources"},{"path":"09-backbone","file":"08-no-images"},{"path":"09-backbone","file":"09-autoxhr-after-load"},{"path":"09-backbone","file":"10-autoxhr-overlapping"},{"path":"09-backbone","file":"11-autoxhr-trigger-additional"},{"path":"09-backbone","file":"12-autoxhr-trigger-additional-after-delay"},{"path":"09-backbone","file":"13-autoxhr-disabled"},{"path":"09-backbone","file":"14-autoxhr-before-page-load"},{"path":"09-backbone","file":"15-delayed-boomerang"},{"path":"09-backbone","file":"16-autoxhr-after-load-with-mutation"},{"path":"17-usertiming","file":"00-usertiming-none"},{"path":"17-usertiming","file":"01-usertiming-basic"}]
```

Comment:
is there any reason we can't pretty-print this json so it's easier to read in code reviews?